src/client/mds_client.cpp

```diff
@@ -945,7 +958,7 @@ LIBCURVE_ERROR MDSClient::GetOrAllocateSegment(bool allocate,
         }
         return LIBCURVE_ERROR::OK;
     };
-    return rpcExcutor.DoRPCTask(task, IOPathMaxRetryMS);
+    return rpcExcutor.DoRPCTask(task, metaServerOpt_.mdsMaxWaitMs);
```

Comment:
I think the previous `IOPathMaxRetryMS` is more meaningful, so you can rename this to `metaServerOpt_.maxRetryMsInIOPath`, and rename the corresponding item in configure file.
Also, replace all `IOPathMaxRetryMS`  with `metaServerOpt_.maxRetryMsInIOPath`.

Reply:
done.